package.json

```diff
@@ -6,8 +6,10 @@
   "scripts": {
     "start": "electron .",
     "build": "rm -rf build && webpack --mode production",
```

Comment:
adding `--config webpack.common.js` resolves the `npm run build issue`

Reply:
Adding that change in now, also displaying the splash page for 1.2 seconds.